modules/data.atmosphere/R/download.raw.met.module.R

```diff
@@ -91,7 +91,8 @@
       method = input_met$method,
       pattern = met,
       dbparms=dbparms,
-      ensemble = ensemble
+      ensemble = ensemble,
+      Rbinary = "R"
```

Comment:
So it looks like the general pattern you've followed is that anywhere `convert.inputs` is called you've hardcoded in `Rbinary = "R"`. The result of this is that if someone passes in a non-standard Rbinary path in via the `settings` object that this path will never get used, which is going to break real-world workflows even if it doesn't break the Github Actions.

Reply:
It seems the existing `host` argument is likely to be passed the entire `settings$host` section, and therefore if Rbinary is set in `settings` I bet it will be accessible inside `convert.inputs` as host$Rbinary`. Is that a reliable enough assumption that we can eliminate the new `Rbinary` argument and read it from `host`?

Reply:
Doh, I didn't realize that `host` was an argument! If that's the case, then yes, we can drop Rbinary as an argument and just replace the original Rbinary line of code
```
Rbinary <- ifelse(!exists("settings") || is.null(settings$host$Rbinary),"R",settings$host$Rbinary)
```
with something like
```
Rbinary <- ifelse(is.null(host$Rbinary),"R",host$Rbinary)
```

Reply:
@Its-Maniaco I think this is the right way to go -- can you undo adding the `Rbinary` argument and implement @mdietze's proposal instead? 
Sorry for the design change! The fact that none of us remembers what all the arguments to `convert.input` do seems like good evidence it has too many parameters :wink: